test/components/views/rooms/RoomHeader-test.tsx

```diff
@@ -200,6 +200,10 @@ describe("RoomHeader", () => {
     });
 
     it("opens the notifications panel", async () => {
+        jest.spyOn(SettingsStore, "getValue").mockImplementation((name: string) => {
```

Comment:
There should be a small  test with `feature_notifications = false`.

Reply:
All the other tests have feature notifications false as it's the default. I will not be adding more cases